### ISSUE TITLE
Coverage directory created in ./web

### ISSUE THREAD

**Issue by jderrough** [redwoodjs/redwood#3413]
With a fresh Redwood blog (`yarn create redwood-app ./redwoodblog`):
1. running `yarn rw test --coverage` creates the coverage directory in `./web`
2. running `yarn rw test api --coverage` or `yarn rw test web --coverage` creates the coverage directory in the project's root directory
3. running `yarn rw test --coverage --coverageDirectory=./coverage` is equivalent to (1)
4. as a workaround, running ``yarn rw test --coverage --coverageDirectory=`pwd`/coverage`` creates the coverage directory in the project's root directory

**Comment by thedavidprice** [comment]
First off, well done using the test suite and considering coverage! Although we're trying to ensure that everyone loves testing as much as we do, we're not quite there yet.

But that behavior isn't exactly consistent. I'm guessing it has to do with the way we need to set `cwd` at different steps (and per options) in the test command:
- https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/test.js

Where do you expect the coverage directory (or directories) to be? 

A simple solution could be to set --coverageDirectory within the command. Otherwise, the command itself might need to be re-architected, which I'm not sure would be a near-term project.

**Comment by jderrough** [comment]
> Where do you expect the coverage directory (or directories) to be?

The current defaults are fine to me. One can get consolidated results by running tests for both sides at once, or separate results by running tests for each side, and be able to specify the directory using `--coverageDirectory`.

It contradicts the documentation that running `yarn rw test --coverage` or `yarn rw test --collect-coverage` sends the consolidated results to `<rootDir>/web/coverage` rather than `<rootDir>/coverage`, when running tests specifying a side does.

**Comment by thedavidprice** [comment]
> The current defaults are fine to me.

Understood.

> It contradicts the documentation ...

Are you referring to Jest documentation? I'm not aware that we mention this in Redwood Docs.

Regardless, agreed that's unexpected behavior as `<rootDir>/coverage` is expected. Especially given the behavior of number 2 in your OP.

**Comment by jderrough** [comment]
> Are you referring to Jest documentation? 

I was referring to [Redwoord's CLI documentation](https://redwoodjs.com/docs/cli-commands#test):
```
--collectCoverage

Show test coverage summary and output info to coverage directory in project root.
See this directory for an .html coverage report
```

**Comment by thedavidprice** [comment]
This will be resolved by #3413
